### PR TITLE
 Changes to use JSON recipe files #161

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes to use JSON recipe files #161

### CODE REVIEW

**Comment by joachimmetz** [comment]
@tomchop I've updated the PR, I've also cleaned up the dftimewolf tool a bit in this PR. Next steps there would be add better error reporting and tests

File: dftimewolf/cli/dftimewolf_recipes.py [hunk 1]
```diff
@@ -68,92 +55,182 @@
...
-      recipe['name'], len(recipe['modules'])))
+    configuration_file_path = os.path.join(
+        '/', 'usr', 'share', 'dftimewolf', 'dftimewolf.conf')
+    config.Config.load_extra(configuration_file_path)
 
-  print('Setting up modules...')
-  state.setup_modules(args)
-  print('Modules successfully set up!')
+  def ParseArguments(self, arguments):
+    """Parses the command line arguments.
```
Comment (tomchop) [inline dftimewolf/cli/dftimewolf_recipes.py]: This function does more than just parse arguments, namely calls `setup_modules` which already calls logic from each individual module. Maybe rename or extract the `setup_module` call?

Comment (joachimmetz) [reply dftimewolf/cli/dftimewolf_recipes.py]: per conversation, I've changed the function.

**Comment by joachimmetz** [comment]
@tomchop PTAL

**Comment by joachimmetz** [comment]
@tomchop PTAL, review status got reset by force push

**Pull request merged**